react-blog/src/App.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect } from "react";
+import "./App.css";
+import { useSelector } from "react-redux";
+import { fetchPosts } from "./store/actions/posts";
+import "./style/style.css";
+import { useAppDispatch } from "./useDispatch";
+
+const App = () => {
+  const {
+    posts,
+    loading: isLoading,
+    error,
+  } = useSelector((state: any) => state.posts);
+
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch(fetchPosts());
+  }, []);
+
+  if (error) {
+    return (
+      <div>
+        <h1>{error}</h1>
+      </div>
+    );
+  }
+
+  return (
+    <div>
+      <header className="header">
```

Comment:
Try to extract this header into separate component

Reply:
@VVMig OK, I did it in my branch feature/kavaliova-lesson-14 with a separate commit:  (https://github.com/VVMig/TMS.FE-41/pull/12/commits/477b9293b9eff995e40684be7ce0edf887c7c013).